src/bundle/Resources/public/ts/validators/ValidatorManager.ts

```diff
@@ -0,0 +1,30 @@
+import BaseValidator from './BaseValidator';
+
+export interface ValidateReturnType {
+    isValid: boolean;
+    messages: string[];
+}
+
+export default class ValidatorManager<T> {
+    private _validators: BaseValidator<T>[];
+
+    constructor(validators: BaseValidator<T>[] = []) {
+        this._validators = validators;
+    }
+
+    addValidator(validator: BaseValidator<T>): void {
+        this._validators.push(validator);
+    }
+
+    removeValidator(validator: BaseValidator<T>): void {
+        this._validators = this._validators.filter((savedValidator) => savedValidator !== validator);
+    }
+
+    validate(value: T): ValidateReturnType {
+        const errors = this._validators
```

Comment:
same suggestion as in: https://github.com/ibexa/design-system/pull/14/files#r2287934977